USERGUIDE.md

```diff
@@ -16,7 +16,7 @@ All the MiniApp files downloaded by the MiniApp iOS library are cached locally
 
 ## Requirements
 
-This module supports **iOS 11.0 and above**. It has been tested on iOS 11.0 and above.
+This module supports **iOS 13.0 and above**. It has been tested on iOS 13.0 and above.
```

Comment:
this change is already waiting to be pushed on candidate branch

Reply:
Oh ok. So you don't want me to push this `USERGUIDE` change in `candidate` branch ? I'm going to push the `ViewController` changes anyway, let me knowl

Reply:
that's fine, not a big deal just merge it. I can manage the change on my side